app/components/Avatar/AvatarWithPresence.js

```diff
@@ -37,20 +39,28 @@ class AvatarWithPresence extends React.Component<Props> {
       isPresent,
       isEditing,
       isCurrentUser,
+      t,
     } = this.props;
 
     return (
       <>
         <Tooltip
           tooltip={
             <Centered>
-              <strong>{user.name}</strong> {isCurrentUser && "(You)"}
-              <br />
-              {isPresent
-                ? isEditing
-                  ? "currently editing"
-                  : "currently viewing"
-                : `viewed ${distanceInWordsToNow(new Date(lastViewedAt))} ago`}
+              <Trans
+                defaults="<strong>{{ userName }} {{ you }} <br /> {{ action }}</strong>"
```

Comment:
While I realise you _can_ put html in these strings, I don't think it's best practice and certainly stands out in the translation files as being odd.

Any reason we can't just translate the four strings separately here?

"you"
"currently editing"
"currently viewing"
"viewed {{ timeAgo }} ago"

Reply:
Note, the `<strong>` tag should only be around the username

Reply:
The reason of this is because in some languages the order of the words changes.

In this case i guess the order doesn't change so much but i am not sure for asian languages like japanese and chinese .

If a translator wants to change the respective order, he only needs to do it in the translation files and not in the code.

Example:
EN: "You have edited the document successfully"
DE: "Du hast die Datei erfolgreich bearbeitet" -> word translated to EN would be "You have the document successfully edited".

Reply:
For now i fixed the **strong**